### ISSUE TITLE
Creating a `r/sys/config` realm for runtime configuration

### ISSUE THREAD

**Issue by moul** [gnolang/gno#1856]
There is a need to create a `r/sys/config` realm to configure and update blockchain configuration by governance. This would include changing parameters such as VM memory limit, CPU cycle, and parameters that will be read by tendermint2 for some custom modules we'll want to develop later.

As for other gnoland module, the goal would be to use the upcoming event emit system (#575) to alert tm2 of a change once the contract confirms that all the requirements are met (i.e., DAO approved a change).

This realm should be linked with genesis. Right now, configuration is either hardcoded or in the configuration file, but in our case, the goal is for the blockchain to start with minimal config while relying then on this realm. So we should imagine maybe having a shorter genesis and configuration file, while focusing on making it easy to extend this realm with new variables that are then read by tm2.

This issue is to make it clear that this realm should be developed but also to have a reference for existing discussions so we can centralize all the discussions here.

Depends on #575 
Related with https://github.com/gnolang/gno/pull/828#pullrequestreview-1433955121
Related with https://github.com/gnolang/gno/issues/1837#issuecomment-2018493517
Related with #1807

**Comment by moul** [comment]
Will be replaced by #2920. 

However, let's keep the issue open in case we need to write a solid realm.

**Comment by Kouteki** [comment]
@moul what values should be in the config?

**Comment by Kouteki** [comment]
Per discussion with the team, I'm closing this issue - we've done enough for the launch milestone.

Future work will be tracked in https://github.com/gnolang/gno/issues/3271
